src/types.ts

```diff
@@ -341,6 +342,7 @@ export interface CaaSApi_Media_File extends CaaSApi_Media_Base {
 
 export interface CaaSApi_Media_Picture extends CaaSApi_Media_Base {
   mediaType: 'PICTURE'
+  changeInfo?: { revision: number }
```

Comment:
In case of possible props getting added to this later on it might be a good idea to exclude this into a small type to prevent errors since it's used redundantly.

Reply:
This has been refactored and moved to `CaaSApi_Media_Base` as this is used by both `CaaSApi_Media_Picture` and `CaaSApi_Media_File`.